packages/chain/contracts/Eth_broker.sol

```diff
@@ -339,7 +338,7 @@ contract Eth_broker {
             "Curve burn failed"
         );
         // Getting expected ETH for DAI
-        uint256 ethMin = sellRewardDai(_minDaiSellValue);
```

Comment:
By removing this value, the passed in value `_minDaiSellValue` (line 301) is not anymore used (only to emit it in an event). Given that this value is not used, it is not best practice to emit it in the event, so you could just remove it from the function signature and also don't emit it in the event. I understand there might be reasons for backwards compatibility that you keep the function signature and event as is (and that this particular change is not even wanted because it is reverting #7 )